purchase_batch_invoicing/views/purchase_order_view.xml

```diff
@@ -25,7 +25,7 @@
         <field name="inherit_id" ref="purchase.purchase_order_tree"/>
         <field name="arch" type="xml">
             <xpath expr="//field[@name='invoice_status']" position="attributes">
-                <attribute name="attrs"/>
+                <attribute name="invisible"/>
```

Comment:
Is this working? Shouldn't be:

```suggestion
                <attribute name="invisible">0</attribute>
```
?

Reply:
I tested it locally and it worked, but to be sure let do it as you stated, ;-)

Reply:
Well, if it was working no need of changing it. I didn't know for sure. That's why I asked. Any way, we both know for the next time.